### PR TITLE
chore(curriculum): add newspaper article lab

### PR DESCRIPTION
**Pull request by moT01**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Review by Dario-DC — COMMENTED** [review]
I left a couple of comments. Other than that it looks good to me :+1:

**Comment by moT01** [comment]
Thanks for taking a look @Dario-DC - I added your suggestions 👍

File: curriculum/challenges/english/25-front-end-development/lab-newspaper-article/66ba762af611169359d9d369.md [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+# --description--
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab. 
+
+**User Stories:**
+
+1. You should set the root `font-size` of your HTML document to `24px`.
+1. You should have an element with a class of `newspaper` that contains all your other elements.
+1. Your `.newspaper` element should have a `font-size` of `16px` and a font of `Open Sans` with a fallback font of `sans-serif`.
+1. Within your `.newspaper` element, you should have at least seven more elements: one for the newspaper name that has a class of `name`, one for the the date of the article with a class of `date`, one for the headline with a class of `headline`, one for the sub-headline with a class of `sub-headline`, one for the author with a class of `author`, and two paragraphs each with a class of `text`. All of these elements should be filled with your article information.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/lab-newspaper-article/66ba762af611169359d9d369.md:18]:
```suggestion
1. Within your `.newspaper` element, you should have at least seven more elements: one for the newspaper name that has a class of `name`, one for the date of the article with a class of `date`, one for the headline with a class of `headline`, one for the sub-headline with a class of `sub-headline`, one for the author with a class of `author`, and two paragraphs each with a class of `text`. All of these elements should be filled with your article information.
```

**Review by Sembauke — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/25-front-end-development/lab-newspaper-article/66ba762af611169359d9d369.md [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+---
+
+# --description--
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab. 
+
+**User Stories:**
+
+1. You should set the root `font-size` of your HTML document to `24px`.
+1. You should have an element with a class of `newspaper` that contains all your other elements.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/lab-newspaper-article/66ba762af611169359d9d369.md:16]:
```suggestion
1. You should have an element with a class of `.newspaper` that contains all your other elements.
```

File: curriculum/challenges/english/25-front-end-development/lab-newspaper-article/66ba762af611169359d9d369.md [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+1. Your `.newspaper` element should have a `font-size` of `16px` and a font of `Open Sans` with a fallback font of `sans-serif`.
+1. Within your `.newspaper` element, you should have at least seven more elements: one for the newspaper name that has a class of `name`, one for the the date of the article with a class of `date`, one for the headline with a class of `headline`, one for the sub-headline with a class of `sub-headline`, one for the author with a class of `author`, and two paragraphs each with a class of `text`. All of these elements should be filled with your article information.
+1. Your `.name` element should have a `font-size` that's two times the size of the root elements `font-size` and a font of `Times New Roman` with a fallback font of `serif`.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/lab-newspaper-article/66ba762af611169359d9d369.md:19]:
```suggestion
1. Your `.name` element should have a `font-size` that is twice the root element's `font-size` and should use the `Times New Roman` font, with a fallback font of `serif`.
```

File: curriculum/challenges/english/25-front-end-development/lab-newspaper-article/66ba762af611169359d9d369.md [hunk 4]
```diff
@@ -0,0 +1,321 @@
...
+1. Your `.name` and `.author` elements should use CSS to make all its characters uppercase.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/lab-newspaper-article/66ba762af611169359d9d369.md:20]:
```suggestion
1. Your `.name` and `.author` elements should use CSS to make all their characters uppercase.
```

File: curriculum/challenges/english/25-front-end-development/lab-newspaper-article/66ba762af611169359d9d369.md [hunk 5]
```diff
@@ -0,0 +1,321 @@
...
+1. Your `.headline` element should `font-size` that's two times the size of its parent's `font-size` and use CSS to make it bold.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/lab-newspaper-article/66ba762af611169359d9d369.md:21]:
```suggestion
1. Your `.headline` element should have a `font-size` that is twice its parent element's `font-size` and should be bold.
```

File: curriculum/challenges/english/25-front-end-development/lab-newspaper-article/66ba762af611169359d9d369.md [hunk 6]
```diff
@@ -0,0 +1,321 @@
...
+1. Your `.sub-headline` element should have a `font-weight` of `100`, a `font-size` that's `1.5` times the size of its parent's `font-size`, and use CSS to make it italicized.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/lab-newspaper-article/66ba762af611169359d9d369.md:22]:
```suggestion
1. Your `.sub-headline` element should have a `font-weight` of `100`, a `font-size` that is `1.5` times its parent element's `font-size`, and should be italicized.
```

File: curriculum/challenges/english/25-front-end-development/lab-newspaper-article/66ba762af611169359d9d369.md [hunk 7]
```diff
@@ -0,0 +1,321 @@
...
+1. Your `.author` should use CSS to make it bold.
+1. Your `.text` elements should be indented `20px`.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/lab-newspaper-article/66ba762af611169359d9d369.md:24]:
```suggestion
1. Your `.text` elements should have a `text-indent` of `20px`.
```

File: curriculum/challenges/english/25-front-end-development/lab-newspaper-article/66ba762af611169359d9d369.md [hunk 8]
```diff
@@ -0,0 +1,321 @@
...
+1. Your `.text` elements should have a `line-height` that's two times the size of its parent's `font-size`.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/lab-newspaper-article/66ba762af611169359d9d369.md:25]:
```suggestion
1. Your `.text` elements should have a `line-height` that is twice their parent element's `font-size`.
```

File: curriculum/challenges/english/25-front-end-development/lab-newspaper-article/66ba762af611169359d9d369.md [hunk 9]
```diff
@@ -0,0 +1,321 @@
...
+1. The first letter of your `.text` elements should be bold and two times the size of its parent's `font-size`. Hint: use the `::first-letter` selector.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/lab-newspaper-article/66ba762af611169359d9d369.md:26]:
```suggestion
1. The first letter of your `.text` elements should be bold and twice the size of their parent element's `font-size`. Use the `::first-letter` selector for this.
```

**Review by Sembauke — CHANGES_REQUESTED** [review]

**Review by Sembauke — APPROVED** [review]

**Review by Dario-DC — APPROVED** [review]
LGTM :+1:

**Pull request merged**